setup.py

```diff
@@ -70,5 +46,25 @@ def check_python_version():
     ],
     packages=find_packages(exclude=["tests", "tests.*", "tests/*"]),
     include_package_data=True,
-    package_data={'version': [os.path.join('graspy', 'version', 'version.txt')]},  # TODO: #454 Also needs changed by https://github.com/microsoft/graspologic/issues/454,
+    package_data={
+        "version": [os.path.join("graspy", "version", "version.txt")]
+    },  # TODO: #454 Also needs changed by https://github.com/microsoft/graspologic/issues/454
+    install_requires=[
+        "networkx>=2.1",
+        "numpy>=1.8.1",
+        "scikit-learn>=0.19.1",
+        "scipy>=1.4.0",
+        "seaborn>=0.9.0",
+        "matplotlib>=3.0.0",
+        "hyppo>=0.1.3",
+    ],
+    extras_require={
+        "dev": [
+            "black",
+            "mypy",
+            "pytest",
+            "pytest-cov",
+            "sphinx-rtd-theme"
```

Comment:
If some of these are for building docs, all of the packages here need to be listed as well:
https://github.com/microsoft/graspologic/blob/dev/docs/requirements.txt

None of these are GPL. A mix of BSD and MIT.